components/stepper/steps/indicators.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Indicators
+page_title: Indicators
+description: Indicators of the Stepper for Blazor.
+slug: stepper-indicators
+tags: telerik,blazor,stepper,indicators
+published: True
+position: 1
+---
+
+
+# Stepper Indicators
+
+This article explains the configuration of the content that will be rendered in the step indicators. Each step could contain text or icon.
+
+>caption In this article:
+
+* [Indicator Text](#indicator-text)
+* [Indicator Icon](#indicator-icon)
+
+## Indicator Text
```

Comment:
What happens if the indicator text you add is long? Is it supported? If not, tell people in this section to use the label for longer texts.

Consider merging the Labels article into this one, or splitting this one into two separate ones - for icons and for indicators. I am not sure if this is th ebest solution, so maybe one or both of the current articles can better explain the relationship between indicators and labels, and icons and do the necessary cross-linking.

Reply:
I am not sure how to proceed with this feedback, please see my thoughts and advise:
- If long text is used in the indicator text, it currently [overflows the indicator](https://github.com/telerik/blazor/issues/2358#issuecomment-852060502) and is therefore logged as a [bug in the themes repo](https://github.com/telerik/kendo-themes/issues/2553), so I am not sure if I should list it as not supported in the docs
- Labels article is deliberately added as a separate one to distinguish it from the indicators. The relationship between indicators and labels could be seen in the steps>overview#Features (I added some more description to the indicators and labels bullets to make it more clear)

Reply:
Add a sentence on their relaton in the current articles and call it done, if the other case is logged as a bugthere is no point in docmenting it.